### ISSUE TITLE
Broken weather in "At a glance" home widget

### ISSUE THREAD

**Issue by e7d** [amirzaidi/AIDLBridge#5]
Since this Saturday (12/01), the weather usually appearing on the home screen in the "At a glance" widget next to the date isn't working anymore.
After some checking, it is still working on a OnePlus 3 with Android 8.1 but failing on both OnePlus 6T and 5T with Android 9.0.

In the same time, the weather form the "At a glance" widget of the Google application is still working.

Screenshot of my OnePlus 6T (Android 9.0), with the weather missing :
![screenshot_20190114-104620](https://user-images.githubusercontent.com/1412033/51105871-83a62280-17ea-11e9-98b8-2fc059ddbc30.jpg)

Screenshot of my wife's OnePlus 3 (Android 8.1), with weather working :
![screenshot_20190114-104749](https://user-images.githubusercontent.com/1412033/51105882-8acd3080-17ea-11e9-97f1-e520e422b3d3.jpg)

**Comment by piotrek1543** [comment]
@e7d  check https://www.reddit.com/r/GooglePixel/comments/86ptep/where_did_my_weather_thing_on_my_home_screen_go/?utm_source=reddit-android

It should solve your problem.

**Comment by e7d** [comment]
Yes. I did not come back, but I solved this problem by disabling then enabling back Google app, which is probably the same, as all app cache/data was removed in the process.

However, I have another problem since a few weeks. I get an error when i click the weather part of the widget. Instead of opening the weather page of the Google app, I get the message
`App is not installed.`

![Screenshot_20190515-091250](https://user-images.githubusercontent.com/1412033/57755747-f28c7d00-76f1-11e9-82ea-22f1651d6d1d.jpg)

**Comment by ogr3** [comment]
Installed 3.9.1 and the new bridge. Weather link stops working with this version of Google 9.88.7.21.

If i uninstall google updates the link work again

**Comment by Yeiarardiem** [comment]
> Installed 3.9.1 and the new bridge. Weather link stops working with this version of Google 9.88.7.21.
> 
> If i uninstall google updates the link work again

Verified, weather link doesn't work for this Google app version. Long-press for settings in At a glance widget also doesn't work.

**Comment by skoging** [comment]
Weather link works for me, but I get the "App isn't installed" message for flight information. Not sure what it's supposed to link to, as I've never tested it before. 

Samsung S8 (Android 9):
Rootless launcher 3.9.1
Pixel bridge v3
Google 10.4.4.21

**Comment by randyabing** [comment]
After I got my phone fixed at the samsung center, techinician seemed to have installed the latest stock android. So uninstalling Google updates does not work to resolve the missing weather for "at a glance". Any thoughts how to fix it? Thanks!
